### ISSUE TITLE
Question about the final result

### ISSUE THREAD

**Issue by Wakals** [hustvl/GaussianDreamer#33]
Thanks for your work for 3D generation, and I have learned a lot from your project. However, after I followed the README, and optimized 'a fox',
'''
python launch.py --config configs/gaussiandreamer-sd.yaml --train --gpu 0 system.prompt_processor.prompt="a fox" system.sh_degree=3
'''
I got a collapse result. There's noting rendered in ./output/a_fox@.../save/it1200-test.mp4 (the video is pure black) and I cannot open the ./output/a_fox@.../save/it1200-test-color.ply by MeshLab (It warns me that mesh contains 63539 vertices with NAN coords and 0 degenerated faces Corrected.). So may there anything wrong for my operations? I'll appreciate it if you reply!

**Comment by Zhao-Yihao** [comment]
I have met the same problem, the ply file cannot be import to unity engine.

**Comment by Peiqi00** [comment]
I also encountered the same problem, the rendered results .png and .mp4 are both black with no content. May I ask if you have found a solution?

**Comment by Zhao-Yihao** [comment]
Actually I didn't solve directly, but it may be because you did not install the pkg tiny-cuda-nn

**Comment by Wakals** [comment]
> I also encountered the same problem, the rendered results .png and .mp4 are both black with no content. May I ask if you have found a solution?

Actually I have not tried to solve this problem. I find that the coordinates of GS do not update during the training (and sometimes go to NaN) and the loss keeps zero or NaN. What's I have learned from the author(s) is that this issue may be caused by gradient propagation for 3DGS within PyTorch-Lightning framework. And they are trying to solve it.

**Comment by lucker26** [comment]
> 是的，请检查是否安装了 PKG TINY-CUDA-NN

hello，原项目发布的shapE_finetuned_with_330kdata.pth链接失效了。您有新的下载链接吗，谢谢

**Comment by Wakals** [comment]
> yes please check the if pkg tiny-cuda-nn installed

Hi, you mean the pkg in requirements.txt? I guess I have installed it and I am able to find `tinycudann==1.7` in `pip list`, however, I still got the broken results.

**Comment by QZhang2111** [comment]
Hi, I have the same problem, has anyone solved it? Thanks a lot

**Comment by WRY0627** [comment]
Hi, I have the same problem, has anyone solved it? Thanks a lot

**Comment by Wakals** [comment]
> Hi, I have the same problem, has anyone solved it? Thanks a lot

You can try threestudio-gaussiandreamer

**Comment by Wakals** [comment]
> Hi, I have the same problem, has anyone solved it? Thanks a lot

You can try [threestudio-gaussiandreamer](https://github.com/cxh0519/threestudio-gaussiandreamer)!

**Comment by WRY0627** [comment]
> > Hi, I have the same problem, has anyone solved it? Thanks a lot
> 
> You can try [threestudio-gaussiandreamer](https://github.com/cxh0519/threestudio-gaussiandreamer)!
Thanks!!!!!!
